tests/ragger/conftest.py

```diff
@@ -0,0 +1,112 @@
+from dataclasses import dataclass
```

Comment:
This conftest is IMO very important, and if we make improvements on the conftest of a given project we have no mean to distribute the update to other projects

Reply:
Updated with Ragger's template conftests.py